docs/notebooks/verification_gridded_cosmo.ipynb

```diff
@@ -0,0 +1,3097 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "de5d8017",
+   "metadata": {},
+   "source": [
+    "##  Gridded Model Verification\n",
+    "\n",
+    "This script verifies output from a ML-based foundation model versus a\n",
+    "traditional NWP system for the atmospheric system. The defaults set at the top of\n",
+    "this script are tailored to the Alps-Clariden HPC system at CSCS.\n",
+    "- The NWP-model is called COSMO-E and is initialised with the ensemble mean of the analysis. Only surface level data is available in the archive at MeteoSwiss.\n",
+    "- The ML-model is called Neural-LAM and is initialised with the deterministic analysis.\n",
+    "- The Ground Truth is the same deterministic analysis as was used to train the ML-model.\n",
+    "- The boundary data for both models is IFS HRES from ECMWF, where the NWP-model got 6 hourly boundary updates and the ML model 12 hourly.\n",
+    "\n",
+    "For more info about the COSMO model see:\n",
+    "- https://www.cosmo-model.org/content/model/cosmo/coreDocumentation/cosmo_io_guide_6.00.pdf\n",
+    "- https://www.research-collection.ethz.ch/handle/20.500.11850/720460"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0311b50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import cartopy.crs as ccrs\n",
+    "import cartopy.feature as cfeature\n",
+    "import dask\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "from dask.distributed import Client, LocalCluster\n",
+    "from pysteps.verification.salscores import sal  # requires scikit-image\n",
+    "from scipy.stats import wasserstein_distance, kurtosis, skew\n",
+    "from scores.continuous import (\n",
+    "    mae,\n",
+    "    mse,\n",
+    "    rmse,\n",
+    ")\n",
+    "from scores.continuous.correlation import pearsonr\n",
+    "from scores.spatial import fss_2d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a806b47",
+   "metadata": {},
+   "source": [
+    "**--------> Enter all your user settings in the cell below. <--------**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf621a4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### DEFAULTS ###\n",
+    "# This config will be applied to the data before any plotting. The data will be\n",
+    "# sliced and indexed according to the values in this config.\n",
+    "PATH_GROUND_TRUTH = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/cosmo.datastore.zarr\"\n",
```

Comment:
Should this be MDP-processed, or the raw zarr?

Reply:
i adapted the script so that all paths point to a MDP-processed zarr now (or neural-lam predictions)

Reply:
But you can point to any zarr and then simply adapt the cell where the data is loaded. After these cells, make sure that the format is the same for all datasets being used in evaluation. You can see an example of this different preprocessing in the nwp cell for cosmo-e. there are some assertions to check that the user didn't mess up the preprocessing, but feel free to add more if useful.